### PR TITLE
Feature/issue templates

### PR DESCRIPTION
**Pull request by toddysm**
Added bug and feature request templates. We should merge those to main so they can become available for users to file issues. Current experience goes to the discussion board.

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@toddysm The question lists look good to me.
Do you think [create an issue form](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) is more convenient than a text-based issue template? This is a new style of issue template provided by GitHub. 

See an example in the [ORAS issue form](https://github.com/oras-project/oras/issues/new?assignees=&labels=bug&template=bug-report.yaml).

**Comment by toddysm** [comment]
The form looks like a better approach. Let me update the free text to a form and update the PR.

**Comment by FeynmanZhou** [comment]
> The form looks like a better approach. Let me update the free text to a form and update the PR.

@toddysm  Thanks. I could help it if needed.

**Comment by toddysm** [comment]
@FeynmanZhou This is updated... Please take another look

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    id: alternatives
+    validations:
+      required: true
+    attributes:
+      label: "What alternatives have you considered?"
+      description: "A clear and concise description of any alternative solutions or features you've considered."
+  - type: textarea
+    id: context
+    validations:
+      required: true
```
Comment (priteshbandi) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:56]: we should remove this as user might not have any additional context to share.

**Review by priteshbandi — APPROVED** [review]
LGTM with one suggestion

**Review by FeynmanZhou — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yaml (hunk 1 above)
Comment (FeynmanZhou) [reply .github/ISSUE_TEMPLATE/feature-request.yaml:56]: Agree, this can be optional.

**Review by FeynmanZhou — COMMENTED** [review]
LGTM

**Pull request merged**